atlp-rwanda/atlp-pulse-fn
# #Ft-39-ticket-system-continued Add Reply Feature on Ticket System

# PR description
This PR continues `ticket system` by adding replies capabilities.

# Description of tasks that were expected to be completed

- [x] Reply on ticket
- [x] Resolve Ticket
- [x] Display details of a ticket

# How has this been tested?

- Pull this pr in local machine
- Set **BACKEND_URL** to `https://devpulse-backend-pr-108.onrender.com/`
- Signin and go on `dashboard/support`
- Submit a message
- Sign out and signin as **superadmin**
- Go to `dashboard/ tickets`
- Open one ticket and see its details
- Using the form, submit a reply


### Track PR
Trello Card: [39-ticket-system-continued](https://trello.com/c/hWy3Ov8x/39-ticket-system-continued)

#### Screenshoots

![Screenshot (103)](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/55640083/352ea609-6161-4241-ac5a-1d064d4c070d)

![Screenshot (104)](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/55640083/81f55a5a-e1ef-4f9f-b436-43e3ebb49cc6)


![Screenshot (105)](https://github.com/atlp-rwanda/atlp-pulse-fn/assets/55640083/632bee59-1220-404b-962f-868d97ba3262)
